src/RamStatement.h

```diff
@@ -982,152 +974,4 @@ class RamLogSize : public RamRelationStatement {
     std::string message;
 };
 
-#ifdef USE_MPI
```

Comment:
If you're removing MPI, then it would be better in its own commit, own PR, and should remove all the other parts of MPI integration as well.